sites/breadcrumbs/breadcrumbs-fr.html

```diff
@@ -17,3 +17,51 @@
 <ul>
 	<li>Le titre de la page ne dois pas être inclus parmis la liste de fil d'arianne.</li>
 </ul>
+
+<p>Exemple pratique pour le fil d'Ariane avec des données structurées au format RDFa.</p>
+<nav id="wb-bc" property="breadcrumb">
+	<h2>Vous êtes ici:</h2>
+	<div class="container">
+		<ol class="breadcrumb" typeof="BreadcrumbList">
+			<li property="itemListElement" typeof="ListItem">
+				<a property="item" typeof="WebPage" href="https://www.canada.ca/fr.html">
+					<span property="name">Canada.ca</span>
+				</a>
+				<meta property="position" content="1">
+			</li>
+			<li property="itemListElement" typeof="ListItem">
+				<a property="item" typeof="WebPage" href="https://wet-boew.github.io/GCWeb/index-fr.html">
+					<span property="name">GCWeb accueil</span>
+				</a>
+				<meta property="position" content="2">
+			</li>
+		</ol>
+	</div>
+</nav>
+<details class="mrgn-tp-lg">
```

Comment:
Remove the details, but keep the Markup.

But the title that will said, this page breadcrumb should generate the following markup

Reply:
Done. Thanks